content/guides/models/track/config.md

```diff
@@ -175,23 +175,25 @@ run.config.update({"lr": 0.1, "channels": 16})
 ```
 
 ### Set the configuration after your Run has finished
-Use the [W&B Public API]({{< relref "/ref/python/public-api/" >}}) to update your config (or anything else about from a complete Run) after your Run. This is particularly useful if you forgot to log a value during a Run. 
+Use the [W&B Public API]({{< relref "/ref/python/public-api/" >}}) to update a completed run's config. 
 
-Provide your `entity`, `project name`, and the `Run ID` to update your configuration after a Run has finished. Find these values directly from the Run object itself or from the [W&B App UI]({{< relref "/guides/models/track/workspaces.md" >}}):
+You will need to provide your entity, project name and the run's ID, which you can get from the Run object or find in the [W&B App UI]({{< relref "/guides/models/track/workspaces.md" >}}):
```

Comment:
```suggestion
You must provide the API with your entity, project name and the run's ID. You can find these details in the Run object or in the [W&B App UI]({{< relref "/guides/models/track/workspaces.md" >}}):
```
@ngrayluna Do we say "W&B Public API" the whole thing, with all those caps? Similar quesation for W&B App UI.

Reply:
Updated!